### ISSUE TITLE
Login Fails - KeyError "location"

### ISSUE THREAD

**Issue by menaechmi** [Linekio/getmyancestors#43]
Steps to reproduce: 

Try to log in through either the graphical or command line option. 

Current behavior: 

```
Downloading: https://www.familysearch.org/auth/familysearch/login
KeyError
Downloading: https://www.familysearch.org/auth/familysearch/login
KeyError
```
This seems to repeat forever

Expected behavior:
Login and download family tree. I was previously able to do so.

The only generated log seems to be the above cycle of downloading keyerror, but I do get a traceback on cancelling (with keyboard interrupt):

```
Traceback (most recent call last):
  File "AppData\Local\Programs\Python\Python311\Lib\site-packages\getmyancestors-1.0.2rc2-py3.11.egg\getmyancestors\classes\session.py", line 46, in login
    url = r.headers["Location"]
          ~~~~~~~~~^^^^^^^^^^^^
  File "AppData\Local\Programs\Python\Python311\Lib\site-packages\requests\structures.py", line 54, in __getitem__
    return self._store[key.lower()][1]
           ~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'location'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "AppData\Local\Programs\Python\Python311\Scripts\getmyancestors-script.py", line 33, in <module>
    sys.exit(load_entry_point('getmyancestors==1.0.2rc2', 'console_scripts', 'getmyancestors')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "AppData\Local\Programs\Python\Python311\Lib\site-packages\getmyancestors-1.0.2rc2-py3.11.egg\getmyancestors\getmyancestors.py", line 176, in main
    fs = Session(args.username, args.password, args.verbose, args.logfile, args.timeout)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "AppData\Local\Programs\Python\Python311\Lib\site-packages\getmyancestors-1.0.2rc2-py3.11.egg\getmyancestors\classes\session.py", line 27, in __init__
    self.logged = self.login()
                  ^^^^^^^^^^^^
  File "AppData\Local\Programs\Python\Python311\Lib\site-packages\getmyancestors-1.0.2rc2-py3.11.egg\getmyancestors\classes\session.py", line 91, in login
    time.sleep(self.timeout)
KeyboardInterrupt
^C
```

**Comment by jaredrummler** [comment]
Same issue here.

**Comment by pinorana** [comment]
I've got the same problem

**Comment by dmfaria** [comment]
The code is sending GET REQUESTS without the User-Agent header. So familysearch is blocking the script with a response: "Request unsuccessful. Incapsula incident ID:..."

I tried to make a get request with the User-Agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36" and it worked.

**Comment by Linekio** [comment]
Thanks for investigation.
Fixed with 409246df1984d5c4a9bb0aa1fb8c19f4294217eb
